### ISSUE TITLE
An update on Fractal's future development

### ISSUE THREAD

**Issue by allmarkedup** [frctl/fractal#449]
Some of you may be aware that I (Mark) have recently stepped away from actively working on Fractal. In the last few months this has led to a lull in progress on the project with some uncertainty about how to best move it forward in the future. To help mitigate this we put out call for maintainers/developers; however we also realise that the project needs some direction and guidance during this period of transition and beyond.

Happily, with Clearleft's support, Danielle (@dkhuntrods) will soon have some more time to dedicate towards ensuring Fractal's future and ongoing development.

With the assistance of the people who have generously offered their support for this project we hope to build an active community to take part in this development, with Danielle taking on the role of community and development manager.

The first step is to work through the outstanding issues on version 1. Once we have a clearer idea of what structure our community will take, we can begin planning how to approach the stalled work on version 2. We'll post another update shortly with more details on how we intend to approach working through the v1 backlog.

**In the meantime, if anyone is interested in contributing to Fractal’s development in any way please get in touch - we need your help!** 

We appreciate your patience during this time of transition and look forward to getting back to regular releases as soon as we can.

**Mark, Danielle and the Fractal Team**

**Comment by brianjhanson** [comment]
I'm interested in helping out if I can.

**Comment by kylejustice** [comment]
I'm interested in helping to continue and maintain the project.

**Comment by ajilderda** [comment]
Awesome guys! I have no experience in Node.js, but willing to help with the design and front-end side of things.

**Comment by tomdavies** [comment]
I'd also be interested in contributing to Fractal, as we're using it extensively at my work.

**Comment by LeBenLeBen** [comment]
As a frequent user of the tool, I would be happy to help Fractal more forward.

**Comment by dkhuntrods** [comment]
@brianjhanson @kylejustice @ajilderda @tomdavies @risker @LeBenLeBen and anyone else that would like to contribute:

I'm going to start a Slack channel for those interested in maintaining - can you please all let me know your usernames on [Fractal's Slack](https://slack.fractal.build/), if you're on there, and sign up if not?



Thanks!

**Comment by brianjhanson** [comment]
Sure thing @dkhuntrods. I'm `Brian Hanson` over there (very creative username)

**Comment by kylejustice** [comment]
@dkhuntrods Yep! My Fractal Slack handle is just `kylejustice`

**Comment by LeBenLeBen** [comment]
@dkhuntrods I’m `LeBen`

**Comment by risker** [comment]
@dkhuntrods I'm `mihkeleidast` on Slack.

**Comment by jaasum** [comment]
@dkhuntrods I'm `jaasum` on Slack.

**Comment by tomjeatt** [comment]
@dkhuntrods would love to get involved in this. Tom Jeatt on Slack.

**Comment by Chapabu** [comment]
We've just started using Fractal pretty heavily where I work, so I'm more than happy to help where I can. I'm chapabu on Slack :)

**Comment by mikkelrom** [comment]
@dkhuntrods I would like to help as well. `mikkelrom` on Slack 🙂

**Comment by thomaseckhardt** [comment]
@dkhuntrods I'd love to get involved in this too. I'm `thomas` on Slack.

**Comment by joshvickerson** [comment]
I have no experience with Fractal, but it's high on my list of tools to use as my team looks to start creating our own component library. If there's anything I can do to help I'd love to. Joined Slack as `joshvickerson`

**Comment by saneef** [comment]
@dkhuntrods I have been using Fractal extensively in our project. Would love to help. `saneef` on Slack.

**Comment by adrienne** [comment]
I just found this - and in fact I just very recently found Fractal - but I love the tool already and I'm interested in helping if I can be useful. I'm not on the slack yet but I plan to sign up tomorrow and I'll drop my username as soon as I do.

**Comment by tomdavies** [comment]
Hi both, great to have you on board! @welitonderesende I've added you to maintainers channel in slack.  @adrienne let Doug March know via Slack DM that you're there and he'll add you too.

**Comment by armstrongb** [comment]
We're enthusiastic users of Fractal and wherever we can help develop/contribute we will endeavour to do so.

**Comment by adrienne** [comment]
I can't seem to get into the Slack signup? It says payment required.

**Comment by dkhuntrods** [comment]
Hi @adrienne there's an issue with the Slack service we have been using - we're in the process of changing it, but if you send me your email at danielle@clearleft.com, I'll send you an invite.

**Comment by aredridel** [comment]
I'm also quite happy to work on this some! We're going all in on fractal at work and it's fantastic.

**Comment by dkhuntrods** [comment]
Thanks @aredridel. I'm on holiday at the moment but back shortly so will be in touch when I'm back!

**Comment by motivis** [comment]
So when is this all going to begin?

Shouldn't they be writing EVERYTHING in TYPESCRIPT ?

JavaScript will derail and cause a train wreck for the library and the consumers

**Comment by Chapabu** [comment]
@motivis As much as I love TypeScript, there are a lot of factors to take ino account before making a decision like this. The biggest issue for me is that we're an open source project, and migrating wholesale to TypeScript is a large barrier to entry to a lot of people. This includes current maintainers and contributors.

I don't think JavaScript is a "dangerous" language, and personally I think it's foolish to make that assumption without any concrete evidence to back it up. It's FAR from a "no go" language.

Regarding the second part of your comment...I'm not sure what you're getting at, almost to the point where I think you may be commenting on the wrong repository.

To summarise; I think it's unlikely that we'll move to TypeScript - but it's definitely at the very least a conversation topic when we start planning v2.

**Comment by motivis** [comment]
Thanks for the heads up... definitely wrong repo 
if someone is searching for concrete evidence that's a major red flag
I will build my own

**Comment by Chapabu** [comment]
Above comment removed as it didn't contribute to the discussion.

**Comment by buchanora** [comment]
I'd be happy to help out as well. Thinking of adopting fractal at my company.

**Comment by Chapabu** [comment]
I think I'm going to remove this one, as I think it's likely served it's purpose. If anyone is visiting this post-closure, please join the [Fractal Slack](http://fractalize.slack.com) and introduce yourself :)

**Comment by tfrijsewijk** [comment]
Wait - How does the Fractal Slack compare to to the Fractal Discord?

**Comment by tomdavies** [comment]
@tfrijsewijk this is a pretty old issue, hence the mention of Slack. The Fractal slack is no longer used, and we’re using Discord instead as it’s a better fit for community-based chat

**Comment by tfrijsewijk** [comment]
OK, thanks! I'm monitoring the Fractal Discord and just wanted to make sure I'm not missing anything :)
